doc/evaluation.rst

```diff
@@ -0,0 +1,331 @@
+
+PRMSE for true scores is defined similar to :ref:`PRMSE for observed scored<r2>`, but with true score :math:`T` used instead of the observed score :math:`H`: the percentage of variance in true score explained by the system score. 
+
+:math:`PRMSE=1-\frac{MSE(T|M)}{\sigma_T^2}`
+
+:math:`MSE(T|M)` (mean squared error when predicting true score with system score) and :math:`\sigma_T^2` (variance of true score) are estimated from MSE and variance for observed scores with two further changes:
+
+- :math:`\hat{H}` is used instead of :math:`H` to compute :math:`MSE(\hat{H}|M)` and :math:`\sigma_{\hat{H}}^2`. :math:`\hat{H}` is the average of two human scores for each response (:math:`\hat{H_i} = \frac{{H_i}+{H2_i}}{2}`). These evaluations use :math:`\hat{H}` rather than :math:`H` because the measurement errors for each rater are assumed to be random and thus partially cancel out making the average :math:`\hat{H}` closer to true score :math:`T` than :math:`H` or :math:`H2`. 
+
+- To compute estimates for true scores, the values for observed scores are adjusted for **variance of measurement errors** (:math:`\sigma_{e}^2`) in human scores defined as:
+
+:math:`\sigma_{e}^2 = \frac{1}{2 \times N_2}\sum_{i=1}^{N_2}{(H_{i} - H2_{i})^2}`
+
+In a simple case where **all responses are double-scored**, the **mean squared error** when predicting true score with system score :math:`MSE(T|M)` is estimated as:
+
+:math:`MSE(T|M) = MSE(\hat{H}|M)-\frac{1}{2}\sigma_{e}^2`
+
+The **variance of true score** (:math:`\sigma_T^2`) is estimated as: 
+
+:math:`\sigma_T^2 = \sigma_{\hat{H}}^2 - \frac{1}{2}\sigma_{e}^2`
+
+The PRMSE formula implemented in RSMTool is more general and allows for both all responses to be double-scored and **only subset of responses to be double-scored**. This formula uses the same computation for :math:`\sigma_{e}^2` but more complex formulas for :math:`MSE(T|M)` and :math:`\sigma_T^2`. The formulas were derived to ensure consistent results regardless of what percentage of data was double-scored. 
+
+
+:math:`MSE(T|M) = \frac{\sum_{i=1}^{N}{c_i(\hat{H_i} - M_i)^2} - N\sigma_{e}^2}{N_1+2N_2}`
+
+:math:`\sigma_T^2=\frac{\sum_{i=1}^{N}{c_i(\hat{H}_i - \bar{\hat{H}})^2}-(N-1)\sigma_{e}^2}{(N-1) + \frac{N_2(N_1+2N_2-2)}{N_1+2N_2}}`
+
+where 
+
+* :math:`C_i=1` or 2 depending on the number of human scores observed for individual 𝑖.
+
+* :math:`\hat{H}` is the average of two human scores :math:`\hat{H_i} = \frac{{H_i}+{H2_i}}{2}` when two scores available or :math:`\hat{H_i} = H_i` when only one score is available. 
+
+* :math:`N_1` is the number of responses with only one human score available (:math:`N_1+N_2=N`)
+
+PRMSE is computed using :ref:`rsmtool.prmse_utils.compute_prmse <prmse_api>`
+
+.. note::
+
+	The formula assigns higher weight to discrepancies between system scores and human score when human score is the average of two human scores than when the human score is based on a single score.
```

Comment:
```suggestion
	The PRMSE formula assigns higher weight to discrepancies between system scores and human scores when the latter are averages than when they are individual scores.
```

Reply:
I think spelling out what the averages and individual scores are makes it clearer. What is it about the original wording that made you want to edit?

Reply:
Sure, if you prefer that's fine too. I'd just make sure that "human score" was appropriately pluralized to agree with "system scores". Or the other way around.

Reply:
I pushed a suggested edit.